packages/gatsby/src/utils/api-node-docs.js

```diff
@@ -111,6 +128,18 @@ exports.onCreateNode = true
 exports.onCreatePage = true
 
 /**
+ * Called when a new layout is created. This extension API is useful
+ * for programmatically manipulating layouts created by other plugins
+ */
+exports.onCreateLayout = true
+
+/**
+ * Called when a new component is created. This is used internally to track
```

Comment:
This doesn't seem needed since we already have `onCreateLayout` and `onCreatePage`.